### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by bartv**
# Description

* Short description here *

closes *Add ticket reference here*

# Self Check:

- [ ] Attached issue to pull request
- [ ] Changelog entry

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 openapi-spec-validator==0.2.9
-PyJWT==1.7.1
+PyJWT==2.0.1
```
Comment (arnaudsjs) [inline requirements.txt:2]: On `inmanta-core` this dependency is still at 1.7.1. Is this one excluded from version bumps?

File: requirements.txt [hunk 2]
```diff
@@ -15,12 +15,12 @@ pyyaml==5.4.1
...
 texttable==1.6.3
 tornado==6.1
-asyncpg==0.21.0
+asyncpg==0.22.0
 jinja2==2.11.3
 psutil==5.8.0
 pydantic==1.7.3
 typing_inspect==0.6.0
-importlib_metadata==2.1.0
+importlib_metadata==3.4.0
```
Comment (arnaudsjs) [inline requirements.txt:23]: on `inmanta-core` this one is still at 2.1.0?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (bartv) [reply requirements.txt:2]: I think this is recent bump then

File: requirements.txt (hunk 2 above)
Comment (bartv) [reply requirements.txt:23]: I think this is recent bump then

**Comment by bartv** [comment]
I will try these versions in a branch on core to be certain

File: requirements.txt (hunk 2 above)
Comment (arnaudsjs) [reply requirements.txt:23]: No. Version 2.1.0 was released on the 24th of november.

**Comment by bartv** [comment]
Testing it here. Might indeed be that they where "snoozed"
https://github.com/inmanta/inmanta-core/pull/2682

File: requirements.txt (hunk 1 above)
Comment (arnaudsjs) [reply requirements.txt:2]: This one is ignored: https://github.com/inmanta/inmanta-core/pull/2609. Can I resume the bumps?

Comment (bartv) [reply requirements.txt:2]: Lets wait for the test PR I linked. When we bump it, it should resume.

**Comment by bartv** [comment]
Both are giving issues, so lets use the versions of core.

**Review by arnaudsjs — APPROVED** [review]

**Pull request merged**